### PR TITLE
Keras Layer docs 

### PR DESCRIPTION
**Pull request by Deliadd**
Tasks:
init   the doc structure
./ELU.scala
./UpSampling2D.scala
./SparseDense.scala
./MaxPooling3D.scala
./LocallyConnected1D.scala
./Reshape.scala
./MaxoutDense.scala
./Convolution1D.scala
./Highway.scala

All finished.

### CODE REVIEW

**Comment by Deliadd** [comment]
Jenkins is successful. ID is 1485.
Here is the link :http://10.239.47.211:18888/view/ZOO-PR/job/ZOO-PR-Validation/1485/

File: docs/docs/KerasStyleAPIGuide/Layers/advanced-activation.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+```
+**Python:**
+```python
+ELU(alpha=1.0, input_shape=None, name=None)
+```
+
+**Parameters:**
+
+* `alpha`: Scale for the negative factor. Default is 1.0.
+* `inputShape`: Only need to specify this argument when you use this layer as the first layer of a model. For Scala API, it should be a [`Shape`](../keras-api-scala/#shape) object. For Python API, it should be a shape tuple. Batch dimension should be excluded.
```
Comment (hkvision) [inline docs/docs/KerasStyleAPIGuide/Layers/advanced-activation.md]: does the link to Shape work?

File: docs/docs/KerasStyleAPIGuide/Layers/advanced-activation.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+**Scala example:**
+```scala
+import com.intel.analytics.zoo.pipeline.api.keras.layers.ELU
+import com.intel.analytics.zoo.pipeline.api.keras.models._
```
Comment (hkvision) [inline docs/docs/KerasStyleAPIGuide/Layers/advanced-activation.md]: change _ to Sequential?

File: docs/docs/KerasStyleAPIGuide/Layers/convolutional.md [hunk 3]
```diff
@@ -0,0 +1,288 @@
...
+
+**Parameters:**
+
+* `nbFilter`: Number of convolution filters to use.
+* `filterLength`: The extension (spatial or temporal) of each filter.
+* `init`: String representation of the initialization method for the weights of the layer. See [here](initialization/#available-initialization-methods) for available initialization strings. Default is 'glorot_uniform'.
+* `activation`: String representation of the activation function to use. See [here](activation/#available-activations) for available activation strings. Default is null.
+* `borderMode`: Either 'valid' or 'same'. Default is 'valid'.
+* `subsampleLength`: Factor by which to subsample output. Integer. Default is 1.
+* `wRegularizer`: An instance of [Regularizer](../../APIGuide/Regularizers/), (eg. L1 or L2 regularization), applied to the input weights matrices. Default is null.
```
Comment (hkvision) [inline docs/docs/KerasStyleAPIGuide/Layers/convolutional.md]: link issue?

**Pull request merged**